docs/decommission.rst

```diff
@@ -0,0 +1,80 @@
+Decommission SecureDrop
+=======================
+
+The following steps will guide you through the decommissioning of your
+SecureDrop instance.
+
+#. **Put a notice in advance on your landing page to inform sources that your
+   instance will soon be retiring.**
+   You may want to direct them to other secure methods of contacting you.
+#. **Locate and create an inventory of all your hardware.**
```

Comment:
This is a great idea, and I wonder if we should also suggest this as part of the install process as well (this could be something tracked on the admin persistent volume)

Reply:
Filed separate docs issue for myself in #5275